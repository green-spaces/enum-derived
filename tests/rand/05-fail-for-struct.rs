use enum_derived::Rand;

#[derive(Rand, PartialEq, Eq, Hash, Debug)]
pub struct Hello {
    world: String,
}

fn main() {}
