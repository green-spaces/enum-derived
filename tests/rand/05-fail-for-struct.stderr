error: proc-macro derive panicked
 --> tests/rand/05-fail-for-struct.rs:3:10
  |
3 | #[derive(Rand, PartialEq, Eq, Hash, Debug)]
  |          ^^^^
  |
  = help: message: Only enums with unit-like variants are supported
